rules/optimize-android-builds-and-start-up-times/rule.md

```diff
@@ -10,18 +10,114 @@ authors:
   url: https://ssw.com.au/people/adam-cogan
 - title: Matt Goldman
   url: https://ssw.com.au/people/matt-goldman
+- title: Anton Polkanov
+  url: https://ssw.com.au/people/anton-polkanov
 related: []
 redirects:
 - do-you-optimise-your-android-builds-and-start-up-times
-
 ---
 
-Whereas iOS enforces AOT (ahead of time) compilation, Android supports (and uses by default) JIT (just in time) compilation, but AOT can be enabled on Android to improve performance.
+.NET MAUI provides several ways to optimize an Android application. Some of them complement each other while others can be mutually exclusive.
+It's important to understand what options you have at hands and how they affect your application.
 
 <!--endintro-->
 
-Xamarin also provides several other options for optimizing Android builds and improving performance. Additionally, you can use Android's new bundle format (rather than apk) so that smaller builds can be targeted to individual hardware specifications.
+## Intro
+
+When it comes to app optimizations, developers usually try to strike the right balance between:
+* app build time
```

Comment:
Hi @AntPolkanov 

I would make the 1st word of these list items...  "app" in uppercase "App"

Reply:
Done